src/components/deputation-modals/ReadOnlyTextField.tsx

```diff
@@ -0,0 +1,26 @@
+import { Input } from '@/components/ui/input';
+
+export function ReadonlyTextField({
+  id,
+  label,
+  value,
+}: {
+  id: string;
+  label: string;
+  value: string;
+}) {
+  return (
+    <fieldset className="flex flex-col md:flex-row mb-2">
+      <label className="block w-1/2 text-sm" htmlFor={id}>
+        {label}
+      </label>
+      <Input
+        className="border-none h-auto p-0 text-sm font-semibold"
+        id={id}
+        value={value}
+        type="text"
+        readOnly
+      />
```

Comment:
Is it worth moving trying to nest the input under the label to auto-associate them without the need for an explicit ID?